setup.py

```diff
@@ -34,46 +33,59 @@
 
 current_os = platform.system()
 if current_os == "Windows":
-    extra_compile_args = ['/std:c++17', '/O2', '/arch:AVX512']
+    # '/arch:AVX512' or '/arch:AVX2'
```

Comment:
why did you take the `arch:AVX512` out? Maybe expand the comment somewhat...

Reply:
I read the compilation instructions in https://github.com/kimwalisch/libpopcnt - turns out the flags are not required as the CPU support checks are done at runtime.